### PR TITLE
Feature/docker

### PR DESCRIPTION
**Pull request by checche**
I edited Dockerfile and docker-compose.yml.  
Just "docker-compose up" will create a django container with the drf-logger and tests directories mounted in the appropriate places.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FROM python:3.7-slim
+WORKDIR /workdir
+RUN apt-get -y update
+ENV LANG en_US.utf8
+COPY requirements.txt /workdir
+COPY requirements-dev.txt /workdir
+RUN pip install -r requirements.txt -r requirements-dev.txt && \
```
Comment (yutayamazaki) [inline Dockerfile]: I want to install drf-logger too into this docker image used in example_project. So, how about add a new line `python setup.py develop` to install a latest drf-logger package.

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+version: '3'
+
+services:
+  django:
+    build: .
+    image: dev_drf_logger
+    ports:
+      - "8000:8000"
+    volumes:
+      - ./example_project:/workdir
```
Comment (yutayamazaki) [inline docker-compose.yml]: What's the difference between `- ./example_project:/workdir` and `- ./drf_logger:/workdir/drf_logger`? I think it's reasonable to write `- ./example_project:/workdir/example_project`

Comment (checche) [reply docker-compose.yml]: This docker-compose.yml changes the directory structure in the docker container to execute Django while importing drf-logger in this repository.  
The reason for writing `-./example_project:/workdir` is to shorten redundant directory paths.

File: Dockerfile (hunk 1 above)
Comment (checche) [reply Dockerfile]: Can I import drf-logger in the working branch by using setup.py? If so, I'll fix it immediately

Comment (yutayamazaki) [reply Dockerfile]: Yes, you can :)

File: docker-compose.yml (hunk 2 above)
Comment (yutayamazaki) [reply docker-compose.yml]: I see. The point is that you want to use drf-logger as a python module , not as a pre-installed package, right? If so, it's no need to modify Dockerfile.

**Comment by yutayamazaki** [comment]
@checche 
If [this comment](https://github.com/yutayamazaki/drf-logger/pull/4#discussion_r387648763) is right, I would like to merge this PR!

**Comment by checche** [comment]
@yutayamazaki 
Using `python setup.py develop`, I could make Dockerfile and docker-compose.yml more concise .  
File editing in `/drf_logger` seems to be immediately reflected in Django app (`/example_project`).

**Pull request merged**